opm/simulators/wells/MultisegmentWellSegments.cpp

```diff
@@ -262,7 +262,7 @@ computeFluidProperties(const EvalWell& temperature,
                                 "rv {} and pressure {}. "
                                 "Continue as if no dissolution (rs = 0) and "
                                 "vaporization (rv = 0) for this connection.",
-                                d, well_.name(), rv, seg_pressure);
+                                d, well_.name(), rs, rv, seg_pressure);
```

Comment:
Sorry!  I should have caught that whilst reviewing #4337...

Reply:
On the bright side, this branch is not executed very often. :sweat_smile: